ecosystem.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     },
     {
       name: 'ethereum-node',
-      script: 'hardhat node --config test/hardhat.config.ts',
+      script: 'hardhat node --config hardhat.config.ts',
```

Comment:
It may be possible to remove the `--config hardhat.config.ts` since it's in the root now, but not a big deal either way.